src/v3/__mocks__/okta.js

```diff
@@ -10,27 +10,12 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { IdxStatus } from '@okta/okta-auth-js';
-import { FunctionComponent, h } from 'preact';
+import Bundles from '../../util/Bundles';
 
-type MessageProps = {
-  type: IdxStatus,
-  message: string,
-  // eslint-disable-next-line react/require-default-props
-  i18n?: string | null,
+module.exports = {
+  ...jest.requireActual('okta'),
```

Comment:
Can test still work if just mock the `loc` function? if possible, let's not over depending on Courage.

Reply:
Tests are failing because some uses the `_` module, I was going to mock that as well but decided to do this as a shortcut, I can circle back on this and mock all of these modules that are used, I first want to get a green bacon then understand how wide spread it is to mock them all. I know things like `_.map` and others are used and cause tests to fail.